app/src/main/java/edu/nextstep/camp/calculator/MainActivity.kt

```diff
@@ -4,88 +4,49 @@ import android.os.Bundle
 import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import edu.nextstep.camp.calculator.databinding.ActivityMainBinding
+import edu.nextstep.camp.common.UiText
 import edu.nextstep.camp.domain.Calculator
 import edu.nextstep.camp.domain.Expression
-import edu.nextstep.camp.domain.Operator.*
+import edu.nextstep.camp.domain.Operator
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity(), MainContract.View {
     private lateinit var binding: ActivityMainBinding
-    private val calculator = Calculator()
-    private var expression = Expression.EMPTY
+    private val presenter = MainPresenter(this, Calculator())
```

Comment:
```suggestion
    private val presenter: MainContract.Presenter = MainPresenter(this, Calculator())
```
타입을 인터페이스 타입으로 변경하게 되면, 해당 인터페이스를 구현한 모든 객체를 사용할 수 있어서, 코드를 좀 더 유연하게 작성할 수 있어요.
인터페이스 타입으로 변경해보면 어떨까요? (이하 관련 내용 동일)